haystack/components/builders/dynamic_chat_prompt_builder.py

```diff
@@ -69,37 +63,29 @@ class DynamicChatPromptBuilder:
     'total_tokens': 238}})]}}
     ```
 
-    The primary advantage of using DynamicChatPromptBuilder is showcased in the examples provided above.
-    DynamicChatPromptBuilder allows dynamic customization of prompt messages without the need to reload or recreate the
-    pipeline for each invocation.
-
-    In the example above, the first query asks for general information about Berlin, and the second query requests
-    the weather forecast for Berlin in the next few days. DynamicChatPromptBuilder efficiently handles these distinct
-    prompt structures by adjusting pipeline run parameters invocations, as opposed to a regular PromptBuilder, which
-    would require recreating or reloading the pipeline for each distinct type of query, leading to inefficiency and
-    potential service disruptions, especially in server environments where continuous service is vital.
-
     Note that the weather forecast in the example above is fictional, but it can be easily connected to a weather
     API to provide real weather forecasts.
     """
 
     def __init__(self, runtime_variables: Optional[List[str]] = None):
         """
-        Initializes DynamicChatPromptBuilder with the provided variable names. These variable names are used to resolve
-        variables and their values during pipeline runtime execution. For example, if `runtime_variables` contains
-        `documents`, your instance of DynamicChatPromptBuilder will expect an input called `documents`.
+        Initializes DynamicChatPromptBuilder with the provided variable names.
+
+        These variable names are used to resolve variables and their values during pipeline runtime execution.
```

Comment:
This line and the one below it can probably integrated into the docstring for the `runtime_variables` parameter.